book/src/help_vm_move.md

```diff
@@ -0,0 +1,115 @@
+# Validator Manager Move
+
+```
+Uploads validators to a validator client using the HTTP API. The validators are defined in a JSON file which can be
+generated using the "create-validators" command. This command only supports validators signing via a keystore on the
+local file system (i.e., not Web3Signer validators).
+
+USAGE:
+    lighthouse validator_manager move [FLAGS] [OPTIONS] --dest-vc-token <PATH> --dest-vc-url <HTTP_ADDRESS> --src-vc-token <PATH> --src-vc-url <HTTP_ADDRESS>
+
+FLAGS:
+        --disable-log-timestamp          If present, do not include timestamps in logging output.
+        --disable-malloc-tuning          If present, do not configure the system allocator. Providing this flag will
+                                         generally increase memory usage, it should only be provided when debugging
+                                         specific memory allocation issues.
+    -h, --help                           Prints help information
+        --log-color                      Force outputting colors when emitting logs to the terminal.
+        --logfile-compress               If present, compress old log files. This can help reduce the space needed to
+                                         store old logs.
+        --logfile-no-restricted-perms    If present, log files will be generated as world-readable meaning they can be
+                                         read by any user on the machine. Note that logs can often contain sensitive
+                                         information about your validator and so this flag should be used with caution.
+                                         For Windows users, the log file permissions will be inherited from the parent
+                                         folder.
+        --stdin-inputs                   If present, read all user inputs from stdin instead of tty.
```

Comment:
Just noting here that the `--stdin-inputs` flag is hidden on Windows and thus if `make cli` is ever run on Windows, this line will be removed and will have to be manually reverted.

Reply:
We could have a makefile target that builds Lighthouse in a Docker image and then generates the CLI text from there? Could be a bit of a hassle though

Reply:
I'm actually a fan of this idea. This would also solve some other undesirable behaviour where if we already have an existing binary in `./target/release` or `./target/debug`, it will just use that, even if there are new commits on your branch (you have to ensure you run `make` again).

It also means if there are any errors during execution, temp files will always be removed.

The downside would be that now, every single time you run `make cli`, you have to build the binary from scratch.
And we would need to pipe through error codes and the new files out of the Docker container.

But I think if we don't do something like this, we'll end up needing to create exceptions every time the Windows CLI help text diverges from the Linux one.